packages/styled-components/src/components/primitives/Image/Image.styled.ts

```diff
@@ -1,9 +1,9 @@
+import { StyledCssProps } from 'components/Components.styled.types';
 import styled from 'styled-components';
+import { customCss } from 'utils/customCss';
 
-export const StyledSource = styled.source`
+export const StyledImage = styled.img<StyledCssProps>`
   user-select: none;
-`;
 
-export const StyledImage = styled.img`
-  user-select: none;
+  ${(props) => customCss(props)}
```

Comment:
this approach is no longer valid, check this PR https://github.com/cmpsr/composer/pull/187